pcb/rules/rule.py

```diff
@@ -4,16 +4,27 @@ class KLCRule(object):
     """
     A base class to represent a KLC rule
     """
-    def __init__(self, module, args, name, description,verbose_message='',fix_verbose_message=''):
+    def __init__(self, module, args, name, description,verbose_message='',fix_message=''):
         self.module = module
         self.name = name
         self.description = description
-        self.verbose_message=verbose_message
-        self.fix_verbose_message=fix_verbose_message
-        self.args=args
+        
```

Comment:
missing `self.args=args` this is required by e.g. rule 7.5 to check whether `--fixmore` was given!